src/network/utilization.rs

```diff
@@ -2,14 +2,14 @@ use crate::network::{Connection, Direction, Segment};
 
 use ::std::collections::HashMap;
 
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
Sure

Reply:
I think cloned is used in `clone_and_reset` + I would say `Debug` never hurts

Reply:
I was referring to the debug... hmm - doesn't it add needless stuff? (I honestly don't know - asking :) )

Reply:
It adds an `impl Debug for X` which is handy to use with `dbg!` among other use cases. The thing is that if you don't impl it for a struct member, you can naturally not have it automatically impl'd for the struct. Maybe personal taste though.

Reply:
Aha - and so it doesn't add anything to the binary unless there's any use of it (with the macro of `{:?}` for example)? So... why don't we derive it for everything?

Reply:
Not sure if it would get optimized out TBH.

Well, there's some influential people that thinks we should, see e.g. Mr. [BurntSushi](https://blog.burntsushi.net/rust-error-handling/)